auction-app-frontend/src/pages/ProductDetails/ProductDetails.jsx

```diff
@@ -102,7 +163,35 @@ const ProductDetails = () => {
               Starts from <span className="price">${ product.startPrice }</span>
             </span>
           </div>
-          <div className="product-bid-details"></div>
+          <div className="product-bid-details">
+            <div className="product-bid-details-item">
+              <span className="item-key">Highest Bid: </span>
+              <span className="item-value">
+                ${ product.highestBid === null ? "0" : product.highestBid }
+              </span>
+            </div>
+            <div className="product-bid-details-item">
+              <span className="item-key">Number of bids: </span>
+              <span className="item-value">{ product.bidsCount }</span>
+            </div>
+            <div className="product-bid-details-item">
+              <span className="item-key">Time left: </span>
+              <span className="item-value">{ timeLeft }</span>
+            </div>
+          </div>
+          { (timeLeft !== "Expired" && userType == "USER" && userId !== product.userId) && (
```

Comment:
`Expired` and `User` can be constants 
And when you compare variable and constant, compare it like this `CONSTANT === VARIABLE`